.circleci/config.yml

```diff
@@ -271,6 +271,32 @@ jobs:
       - codecov/upload: 
           flags: 'test_12,isolation'        
 
+
+
+  test-11_check-citus-upgrade:
+    docker:
+      - image: 'citus/citusupgradetester-11:dev'
+    working_directory: /home/circleci/project     
+    steps:
+      - {attach_workspace: {at: .}}
+      - run: 
+          name: 'Install and test citus upgrade'
+          command: |
+           chown -R circleci:circleci /home/circleci
+           install-and-test-ext --target check-citus-upgrade --citus-pre-tar /install-pg11-citusv8.0.0.tar --citus-post-tar /home/circleci/project/install-11.tar 
```

Comment:
`--citus-post-tar /home/circleci/project/install-11.tar` is very redundant, I wonder if that one can be deduced in the `install-and-test-ext` script based on the PG major version

Reply:
Yeah we can do that.